extensions/wikia/PortableInfoboxBuilder/tests/PortableInfoboxBuilderServiceTest.php

```diff
@@ -24,7 +24,7 @@ public function testTranslationFromData( $data, $expected ) {
 	 * @dataProvider markupTranslationsDataProvider
 	 */
 	public function testTranslationFromMarkup( $markup, $expected ) {
-		$generated_json = json_decode( $this->builderService->translateMarkupToData( $markup ) );
+		$generated_json = json_decode( json_encode( $this->builderService->translateMarkupToData( $markup ) ) );
```

Comment:
comment added where should be: https://github.com/Wikia/app/pull/10064/files#diff-682d483e4e2f7d44e7b9939661b3110eR27
:dancers: